### PR TITLE
DVCSMP-2807 GWWST-77: Z-Wave Metering Switch (Secure) for support for the Fibaro Smart Switch ZW5

### PR DESCRIPTION
**Pull request by dkirker**
This is a carry-over from https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2206

### CODE REVIEW

**Review by mckeed — APPROVED** [review]

File: devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+ *
+ */
+metadata {
+	definition (name: "Z-Wave Metering Switch Secure", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Energy Meter"
+		capability "Health Check"
+		capability "Light"
+		capability "Polling"
```
Comment (mckeed) [inline devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy]: I'd think this device would be hub-polled, so this probably isn't necessary.

**Comment by dkirker** [comment]
@workingmonk This is good for a merge.

File: devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy (hunk 1 above)
Comment (dkirker) [reply devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy]: Removed!

**Comment by workingmonk** [comment]
has this been certified?

**Comment by dkirker** [comment]
@workingmonk Yes, it appears so.

**Comment by workingmonk** [comment]
@dkirker can someone make a definite call on this one? if yes, then i'll go ahead and merge this one

**Comment by workingmonk** [comment]
this has been pending for close to 4-5 months and i am not sure if this is still required. someone would have come looking for this device in prod. that is why want to make sure that this DTH is still required

**Comment by dkirker** [comment]
@workingmonk No one has complained yet because there is an active ticket against the ui meta data (so folks are using self-published). Magnus and I discussed this the other week. We will merge this. I will notify you once it has been certified, again.

**Comment by workingmonk** [comment]
ok cool. ping me on slack when this is ready to merge

**Comment by greens** [comment]
Looks like this has been qualified. @dkirker can you merge?

**Comment by dkirker** [comment]
@greens I just checked, it is. @workingmonk will have to merge, as I lack the right privileges. 😢

**Comment by greens** [comment]
@workingmonk What's the best way for us to also get this commit on master?

**Comment by workingmonk** [comment]
rolled it down, you should have it now

**Pull request merged**